### PR TITLE
Sfm plus add attitude hold (3)

### PR DESCRIPTION
**Pull request by BradMick**

### CODE REVIEW

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/headers/core.hpp [hunk 1]
```diff
@@ -37,6 +37,6 @@
 
 #define HDG_HOLD_SPEED_SWITCH_DECEL   15.433  //30kts GS
 #define HDG_HOLD_SPEED_SWITCH_ACCEL   20.577  //40kts GS
-#define HDG_HOLD_BREAKOUT_VALUE       0.03
-#define VEL_HOLD_BREAKOUT_VALUE       0.06
-#define ATT_HOLD_BREAKOUT_VALUE       0.09
\ No newline at end of file
+#define HDG_HOLD_BREAKOUT_VALUE       0.030
+#define VEL_HOLD_BREAKOUT_VALUE       0.060
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/headers/core.hpp:41]: Please can you remove these unnecessary changes?

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fmc/fn_fmcAttitudeHoldEnable.sqf [hunk 2]
```diff
@@ -23,6 +24,11 @@ if (_heli getVariable "fza_ah64_attHoldActive" == false) then {
     //Attitude hold
     if (_curVel > VEL_HOLD_SPEED_SWITCH_ACCEL) then {
         _heli setVariable ["fza_ah64_attHoldSubMode", "att", true];
+
+        private _curAtt   = _heli call BIS_fnc_getPitchBank;
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fmc/fn_fmcAttitudeHoldEnable.sqf:28]: Why do get the pitch bank, split it out and then rejoin it rather than just using the function output?

**Review by mattysmith22 — CHANGES_REQUESTED** [review]
Looks good, just a couple small refactorings to be made

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fmc/fn_fmcHoldModesDisable.sqf [hunk 3]
```diff
@@ -1,9 +1,9 @@
 params ["_heli"];
 
 //De-activate attitude hold and set the reference back to 0
-_heli setVariable ["fza_ah64_attHoldActive",     false, true];
+_heli setVariable ["fza_ah64_attHoldActive",     false,      true];
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fmc/fn_fmcHoldModesDisable.sqf:4]: Was this intentional?

**Review by mattysmith22 — APPROVED** [review]
Approving, though just double-checking that a change was expected. If not just fix and I will re-approve

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fmc/fn_fmcHoldModesDisable.sqf (hunk 3 above)
Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fmc/fn_fmcHoldModesDisable.sqf:4]: Yup
